source/contribute/development.html.md

```diff
@@ -156,6 +156,24 @@ There is no need to provide individual translations of  `i18n!`-wrapped strings
 The strings will be uploaded to a third-party web service and translated automatically as
 a separate step.
 
+### Adding a language
+
+Managers of the Crowdin project may add new languages to the list of target languages.
+But for these new languages to be visible in the interface as well, some changes need to be made to the code.
+
+First of all, make sure that the corresponding `.po` file is present in `po/plume`, and in `po/plume-front`.
+If it is not, ask someone who has access to the Crowdin API key (like a Crowdin manager or owner) to pull it with the Crowdin CLI tool.
+
+To find a manager, look at the list [in the sidebar of this page](https://crowdin.com/project/plume).
+These person are managers because they were involved enough in the project, and especially in the translation process.
+If you want to become a manager on Crowdin too, just ask us.
```

Comment:
`[ask us](link to our matrix channel)!`

leave out the "just"

(the reasoning here is either tongue in cheek, or cynical: there is no "justice" in software, so no point in using that word, ever)